cypress/cypress/component/Input/BasicInputField.cy.tsx

```diff
@@ -95,6 +95,37 @@ describe("Input.cy.tsx", () => {
     cy.get("@onSubmitSpy").should("be.calledOnceWith", { [name]: randomWord });
   });
 
+  it("validation works for sub-objects", () => {
```

Comment:
i would also change this to nested fiels instead of sub-objects.

Reply:
yes sure, is changed